qiskit_ibm_runtime/sampler.py

```diff
@@ -148,23 +150,16 @@ class SamplerV1(BasePrimitiveV1, Sampler, BaseSampler):
 
     Example::
 
-        from qiskit.circuit import QuantumCircuit, QuantumRegister, ClassicalRegister
+        from qiskit.test.reference_circuits import ReferenceCircuits
```

Comment:
`qiskit.test` no longer exists in 1.0

Reply:
Replaced with explicit code for generating bell circuit - is there a better alternative? Also, we have similar example in `provider_session.py`